api/textsnippetRoutes.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const txtgen = require('txtgen');
+const router = express.Router();
+
+router.post('/generateText' , (req, res) => {
+    let letters = /^[A-Za-z]+$/;
+    const sum = req.body.score;
+    const textsnippet = txtgen.paragraph([sum])
+    if(sum === ""){
+         res.send({error: 'please enter a number'})
+    }
+    else if (sum.match(letters)){
```

Comment:
adjust the line like this:
```js
} else if ... {
```

Usually it's written like this. And fix all the occurrences